src/metadata/metadata.proto

```diff
@@ -20,6 +24,8 @@ message Edit {
     pfs_v2.BranchPicker branch = 3 [(validate.rules).message.required = true];
     // repo targets a repo's metadata.
     pfs_v2.RepoPicker repo = 4 [(validate.rules).message.required = true];
+    // cluster targets the cluster's metadata.
+    ClusterPicker cluster = 5 [(validate.rules).message.required = true];
```

Comment:
Why add this field if no values can be passed?

Reply:
There is actually a message here; an empty message is different from a null message.

This is what `protoc --decode_raw` looks like when no picker is set: 

```
1 {
  10 {
    1 {
      1: "dev"
      2: "true"
    }
  }
}
```

This is what it looks like if you specify `"cluster": {}`:
```
1 {
  5: ""
  10 {
    1 {
      1: "dev"
      2: "true"
    }
  }
}
```